humans/static/javascripts/update_user_form.js

```diff
@@ -5,33 +5,16 @@ $(document).ready(function(){
   $(".label-form-b label").addClass("smallmedium text-darkest");
   $(".checkbox- label").addClass("smallmedium text-darkest");
 
-  $("#tab1").click(function(){
-    //slide up all the link lists
-    $("#section1").show();
-    $("#section2").hide();
-    $("#section3").hide();
-    $("#tab1").addClass("active");
-    $("#tab2").removeClass("active");
-    $("#tab3").removeClass("active");
-  });
-  $("#tab2").click(function(){
-    //slide up all the link lists
-    $("#section2").show();
-    $("#section1").hide();
-    $("#section3").hide();
-    $("#tab2").addClass("active");
-    $("#tab1").removeClass("active");
-    $("#tab3").removeClass("active");
-  });
-  $("#tab3").click(function(){
-    //slide up all the link lists
-    $("#section3").show();
-    $("#section1").hide();
-    $("#section2").hide();
-    $("#tab3").addClass("active");
-    $("#tab1").removeClass("active");
-    $("#tab2").removeClass("active");
+  $(".sett__nav__item").click(function(evt) {
```

Comment:
For some reason this event is not being added to `#tab4` (at least on Firefox)

Reply:
Just tested again, with different browsers and it is working fine. Not sure what was the problem the other time.